dev/PushNotifications/GetRawNotificationEventArgs.h

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#pragma once
+#include "PushNotificationReceivedEventArgs.h"
+#include <winrt/Windows.ApplicationModel.Core.h>
+#include "externs.h"
+
+static PCWSTR c_pushContractId = L"Windows.Push";
+
+using namespace winrt::Windows::ApplicationModel::Core;
+namespace winrt::Microsoft::Windows::PushNotifications
+{
+    static winrt::Windows::Foundation::IInspectable Deserialize(winrt::Windows::Foundation::Uri const&)
+    {
+        auto appProperties = CoreApplication::Properties();
```

Comment:
This pattern looks weird. We are reusing the same code here and below the Wait. Can we condense it to a single Property check call after the WaitForSingleObject?